Maman11/Q2/my_suffix.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#include <string.h>
+#define STRING_LENGTH 127
+
+int suffix(char str[], char c){
+    bool read = false;
```

Comment:
This is good but for uni you cannot use `studbool.h` so try it with a different way. I do want you to have true, false words.

Still this is vary good idea when I code in my job I use the true false of that is included.

Try this and understand the meaning.
`typedef enum {FALSE = 0, TRUE} boolean;`